processing/src/main/java/org/apache/druid/segment/vector/BitmapVectorOffset.java

```diff
@@ -114,6 +124,9 @@ public int getCurrentVectorSize()
   @Override
   public int getStartOffset()
   {
+    if (isContiguous) {
```

Comment:
According to the interface, `getOffsets` is supposed to throw an exception if isContiguous is true, so we should check it there too.

Reply:
added, and tests